app/entrypoint.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
-# shellcheck disable=SC2155
+# shellcheck disable=SC2155,SC2002
 
-set -u
```

Comment:
this change was to make it not error on ` "${DOCKER_PROVIDER,,}"` which is possibly unbound (but actually is not because of the -n sanity)